plugins/source/aws/resources/services/elasticache/gen.hcl

```
@@ -123,4 +123,575 @@ resource "aws" "elasticache" "clusters" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.GlobalReplicationGroup"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  multiplex "AwsAccountMultiplex" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.AccountMultiplex"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountFilter"
+  }
+
+  options {
+    primary_keys = [
+      "arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+}
+
+resource "aws" "elasticache" "replication_groups" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.ReplicationGroup"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = [
+      "arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "auto_minor_version_upgrade" {
+    description = "Auto minor version upgrade."
+  }
+
+  column "global_replication_group_info_global_replication_group_member_role" {
+    rename = "global_replication_group_member"
+  }
+
+  column "global_replication_group_info_global_replication_group_id" {
+    rename = "global_replication_group_id"
+  }
+
+  column "pending_modified_values_auth_token_status" {
+    rename = "pending_auth_token_status"
+    description = "Pending modified auth token status"
+  }
+
+  column "pending_modified_values_automatic_failover_status" {
+    rename = "pending_automatic_failover_status"
+    description = "pending autmatic failover for this redis replication group"
+  }
+
+  # Actually skipping a relation.
+  column "pending_modified_values_log_delivery_configurations" {
+    skip = true
+  }
+
+  column "pending_modified_values_primary_cluster_id" {
+    rename = "pending_primaty_cluster_id"
```

```suggestion
    rename = "pending_primary_cluster_id"
```

Typo in the name here